src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,9 @@
 package core.basesyntax;
 
 public class MainApp {
-
+Machine[] machines = new Machine[] {new Truck(), new Bulldozer(), new Excavator()};
+for (int i = 0;i < machine.length;i++) {
+    System.out.println(machine[i].doWork());
+    System.out.println(machine[i].stopWork());
```

Comment:
The methods `doWork` and `stopWork` are void methods and do not return any value. Therefore, using `System.out.println` is incorrect. You should directly call `machine[i].doWork();` and `machine[i].stopWork();` without `System.out.println`.